test/evergreen.yml

```diff
@@ -2033,6 +2051,64 @@ tasks:
           test_config_filename: configs/reverse_split_stress.txt
           test_name: reverse_split
 
+# Cppsuite perf tests
+  - name: cppsuite-hs-cleanup-default-perf
+    depends_on:
+      - name: compile
+    tags: ["cppsuite-perf-test"]
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite perf test"
+        vars:
+          test_config_filename: configs/hs_cleanup_default.txt
+          test_name: hs_cleanup
+
+  - name: cppsuite-hs-cleanup-stress-perf
+    depends_on:
+      - name: compile
+    tags: ["cppsuite-perf-test"]
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite perf test"
+        vars:
+          test_config_filename: configs/hs_cleanup_stress.txt
+          test_name: hs_cleanup
+
+  - name: cppsuite-operations-test-default-perf
+    depends_on:
+      - name: compile
+    tags: ["cppsuite-perf-test"]
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite perf test"
+        vars:
+          test_config_filename: configs/operations_test_default.txt
+          test_name: operations_test
+
+  - name: cppsuite-operations-test-stress-perf
+    depends_on:
+      - name: compile
+    tags: ["cppsuite-perf-test"]
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite perf test"
+        vars:
+          test_config_filename: configs/operations_test_stress.txt
+          test_name: operations_test
+  # This is a perf test and as such doesn't run under the stress test tag. This name seems excessive
+  # but in order to have a consistent naming system is required, the issue here is that the test
+  # itself has the word "perf" in it.
```

Comment:
```suggestion
  # itself has the word "stress" in it.
```

Reply:
Nah see the test name is "bounded-cursor-perf" the config used is the stressful config so you get "bounded-cursor-perf-stress" then to indicate its a perf test append "perf" and finally "bounded-cursor-perf-stress-perf".

Reply:
Ah gotcha. I thought we were explaining why a test with stress in the name isn't actually a stress test.